### PR TITLE
Reorg sequencer messages if max delay has passed

### PR DESCRIPTION
**Pull request by PlasmaPower**

### CODE REVIEW

File: packages/arb-rpc-node/batcher/sequencerBatcher.go [hunk 1]
```diff
@@ -101,11 +104,19 @@ func NewSequencerBatcher(
...
+	maxDelaySeconds, err := sequencerInbox.MaxDelaySeconds(callOpts)
+	if err != nil {
+		return nil, err
+	}
+
 	feedBroadcaster := broadcaster.NewBroadcaster(broadcasterSettings)
 	err = feedBroadcaster.Start(ctx)
 	if err != nil {
 		logger.Warn().Err(err).Msg("error starting feed broadcaster")
-		return nil, err
```
Comment (hkalodner) [inline packages/arb-rpc-node/batcher/sequencerBatcher.go:108]: Was removing this intentional?

File: packages/arb-rpc-node/batcher/sequencerBatcher.go [hunk 2]
```diff
@@ -395,6 +408,58 @@ func (b *SequencerBatcher) createBatch(ctx context.Context, newMsgCount *big.Int
...
+		if newestMsgCount.Cmp(newMsgCount) > 0 {
+			newerBatchItems, err := b.db.GetSequencerBatchItems(newMsgCount, newestMsgCount)
+			if err != nil {
+				return false, err
+			}
+			batchItems = append(batchItems, newerBatchItems...)
+		}
+
+		var previousSeqBatchAcc common.Hash
+		if prevMsgCount.Cmp(big.NewInt(1)) > 0 {
```
Comment (hkalodner) [inline packages/arb-rpc-node/batcher/sequencerBatcher.go:431]: Why is this greater than 1 instead of greater than 0?

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-rpc-node/batcher/sequencerBatcher.go (hunk 1 above)
Comment (PlasmaPower) [reply packages/arb-rpc-node/batcher/sequencerBatcher.go:108]: No, that got caught in a merge, thanks

File: packages/arb-rpc-node/batcher/sequencerBatcher.go (hunk 2 above)
Comment (PlasmaPower) [reply packages/arb-rpc-node/batcher/sequencerBatcher.go:431]: Interesting, I copied this from deliverDelayedMessages, which was also apparently broken haha. I'll fix them both.

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**